pom.xml

```diff
@@ -9,17 +9,26 @@
 
     <properties>
         <java.version>1.8</java.version>
-        <handlebars-java-version>4.0.3</handlebars-java-version>
+        <handlebars-java-version>4.3.1</handlebars-java-version>
+        <handlebars-markdown-java-version>4.2.1</handlebars-markdown-java-version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <powermock.version>1.5.6</powermock.version>
-        <dp-logging.version>v2.0.0-beta.8</dp-logging.version>
-        <fasterxml.jackson.version>2.15.0</fasterxml.jackson.version>
+        <dp-logging.version>v2.0.0-beta.10</dp-logging.version>
+        <fasterxml.jackson.version>2.16.1</fasterxml.jackson.version>
         <jetty.version>9.4.53.v20231009</jetty.version>
-        <flying-saucer.version>9.1.22</flying-saucer.version>
+        <flying-saucer.version>9.4.0</flying-saucer.version>
     </properties>
 
     <dependencies>
-
+        <dependency>
```

Comment:
I don't think that we need to explicitly require this one here, as it seems to be importing this exact version by default anyway. In other words, if you remove this dependency from here, it still imports it (this exact same version).

Reply:
Test are failing because its missing a org.slf4j.impl.StaticLoggerBinder.class. So that impl of the class is in sl4j-api.

what actually is missing is this:

            <groupId>org.slf4j</groupId>
            <artifactId>slf4j-api</artifactId>
            <version>2.0.7</version>

It should be probably added to dp-logging as dependecy

Reply:
That sounds like the right thing to do 👍